src/modules/common/ToolBar.vue

```diff
@@ -40,7 +40,12 @@
             <v-list-item-title v-text="$t(child.title)" />
           </v-list-item>
         </v-list-group>
-        <v-list-item v-else link>
+        <v-list-item
+          v-else
+          link
+          :to="item.to ? item.to : ''"
+          @click="item.logout ? logout() : null"
+        >
```

Comment:
Não faz sentido ser outro `v-if-else="item.to && item.logout"` ?

Reply:
Done !